nms/app/theme/design-system/Text.tsx

```diff
@@ -9,17 +9,14 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- *
- * @flow strict-local
- * @format
  */
 
 import * as React from 'react';
 import classNames from 'classnames';
 import {makeStyles} from '@material-ui/styles';
 import {typography} from '../default';
 
-const styles = {
+export const typographyStyles = makeStyles(() => ({
```

Comment:
This can be simplified to 
`export const typographyStyles = makeStyles({
    h1: typography.h1,
    ...
```